app/Http/Controllers/UserController.php

```diff
@@ -154,6 +154,31 @@ public function show($id, Request $request)
         }
     }
 
+    /**
+     * Displays the teams the specified user belongs to.
+     *
+     * @param int $id
+     * @param Request $request
+     * @return \Illuminate\Http\JsonResponse
+     */
+    public function showTeams($id, Request $request)
+    {
+        $user = User::findByIdentifier($id)->first();
+        if ($user) {
+            $requestingUser = $request->user();
+            if ($requestingUser->hasRole('admin')) {
```

Comment:
This should be a check for a permission instead of a role. The permission we need here does not exist yet afaik, but we will need to also update the corresponding team api endpoint.

Reply:
// @RossIV

Reply:
As discussed in Slack, see `2018_05_27_220626_add_hidden_teams_permission.php` as an example for permissions for this.

Reply:
Add a permission for users to view team membership and we'll use it for #461 once that's a thing. Add the permission to the admin and officer roles.